pyartifactory/models/User.py

```diff
@@ -23,6 +23,10 @@ class NewUser(BaseUserModel):
     password: SecretStr
 
 
+class User(BaseUserModel):
+    email: Optional[EmailStr] = None
+
+

```

Comment:
What does this new model represent? (Just asking ^^)

Reply:
This is for fixing the update user issue. Indeed, the method update user used to require a NewUser object. However, this object contains a password, which is problematic for update operations. Therefore, I created this new model only for the update operation.